main/models.py

```diff
@@ -444,6 +444,17 @@ class Meta:
         db_table = 'package_files'
 
 
+class PackageSecurity(models.Model):
+    pkg = models.ForeignKey(Package, on_delete=models.CASCADE)
```

Comment:
```
    pkg = models.OneToOneField(Package, on_delete=models.CASCADE)
```

will get you a single `Package.packagesecurity` attribute rather than having to deal with a list/set of one.